src/Form/WebdamConfig.php

```diff
@@ -65,8 +104,24 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#title' => $this->t('Client secret'),
       '#default_value' => $config->get('secret'),
       '#description' => $this->t('API Client Secret to use for API access. Contact the Webdam support team to get one assigned. Note that this field will appear blank even if you have previously saved a value.'),
+      '#required' => TRUE,
     ];
 
+    if (!empty($config->get('username')) && !empty($config->get('password'))) {
```

Comment:
Applied this change in latest commit [9a1e958](https://github.com/mobomo/media_webdam/pull/12/commits/9a1e9586fca7ca638d44d9481e0a752815a34989)